ImageTransform/CommonPart/unit_tests/Unit_Test_CommonPart_lib.cpp

```diff
@@ -0,0 +1,51 @@
+#include <CommonTests.hpp>
+#include <QDebug>
+#include <QDir>
+#include <QFileInfo>
+#include <QImage>
+#include <QPolygonF>
+#include <Transformation.hpp>
+#include <memory>
+
+struct params
+{
+    bool p;
+};
+
+class InterpolationTestWithParams : public testing::TestWithParam<params>
+{
+
+public:
+    using Transform_float =
+        common::Transform<float, QPointF, QRectF, QPolygonF>;
+    using Transform_int = common::Transform<int, QPoint, QRect, QPolygon>;
```

Comment:
This using may be moved to place where template is defined. There is one typedef in place.

Reply:
Done.